### PR TITLE
ignore @ and # sign

### LINKED ISSUE #320: hashtags
<https://forum.languagetool.org/t/ignore-hashtags-user-mentions/3305/14>

### PR DESCRIPTION
**Pull request by lsmith77**
fixes #320

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -431,6 +432,13 @@ async def languagetool_rules(user_request_in: RequestIn):
                         for match in result["matches"]:
                             offset = int(match["offset"])
                             end = offset + int(match["length"])
+                            text = user_request_in.text[offset:end]
+                            # ignore text that starts with @ or #
+                            if text[0:1] in ignore or (
```
Comment (Szarad) [inline app/main.py:437]: I can't wrap my head around why this `or` is needed

Comment (lsmith77) [reply app/main.py:437]: LT actually also does some spell checking of hash tags .. so if you write `#wrld` they will highlight `wrld` so to filter that out, we also need to check if the sign before is `#` or `@`

Comment (Szarad) [reply app/main.py:437]: ah ok, I see now that's for `#`. The test didn't go there, that why I was confused. You didn't want to test `#` as well on one go or another test?

Comment (lsmith77) [reply app/main.py:437]: tweaked the test

**Review by Szarad — APPROVED** [review]
All good now

**Pull request merged**